File: Semana 1/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{

    int height;
    do
    {
        height = get_int("Insert the height: ");
    }
    while (height<1);

    // Print a pyramid of that height
    int counter = 1;
    while (counter <= height) {
        int spaces = height - counter;
        for(int i = 0; i < spaces; i++) {
            printf(" ");
        }
        for(int i = 0; i < counter; i++) {
            printf("#");
        }
        printf("\n");
        counter++;
    }
}

File: Semana 6/mario.py
from cs50 import get_int


height = get_int("Height: ")
wall = "#"
space = " "
while height <1 or height>8:
    height = int(input("Insert the height: "))
for i in range (1, height + 1, 1):
    spaces = height - i

    print((space * spaces) + (wall * i) + "  " + (wall * i))

